critiquebrainz/db/review.py

```diff
@@ -471,15 +471,17 @@ def get_reviews_list(connection, *, inc_drafts=False, inc_hidden=False, entity_i
     result = connection.execute(query, filter_data)
     count = result.fetchone()[0]
     order_by_clause = str()
+    if sort_order == None:
```

Comment:
small thing here - typically in python if you want to compare something against none you can use `if not sort_order`, or if you really want to know that it's None and not an empty string/empty list you can use `if sort_order is None`

Reply:
Understood